src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -146,6 +330,19 @@ static void Fred()
 False
 6
 True
+9
```

Comment:
This should be done for all tests in one pass, so I'm not gonna change it in this PR. Created an issue #10931 to track this.